synapse/types.py

```diff
@@ -421,22 +421,92 @@ class RoomStreamToken:
 
```

Comment:
What's the preference on referring to these tokens? "pagination tokens" or "position tokens"?

Reply:
I've only ever heard them called "pagination tokens".